src/03_modules.py

```diff
@@ -5,27 +5,28 @@
 level operating system functionality.
 """
 
+import os
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
-# YOUR CODE HERE
+print(sys.argv[0])
 
```

Comment:
It looks like you recognized this is a list. To print one per line, you should probably iterate through the list somehow.